app/src/main/java/co/touchlab/droidconandroid/shared/network/dao/NetworkUserAccount.java

```diff
@@ -2,7 +2,7 @@
 /**
  * Created by kgalligan on 4/6/16.
  */
-public class UserAccount
+public class NetworkUserAccount
```

Comment:
These fields are in UserAccount, can we use UserAccount just like how we consolidated Block?